src/analysis/modals/ComputeModal/AzureComputeModal/AzureComputeModal.js

```diff
@@ -64,15 +66,21 @@ export const AzureComputeModalBase = ({
   const hasGpu = () => !!azureMachineTypes[computeConfig.machineType]?.hasGpu;
   // Lifecycle
   useOnMount(() => {
-    const loadCloudEnvironment = _.flow(
-      withErrorReportingInModal('Error loading cloud environment', onError),
-      Utils.withBusyState(setLoading)
-    )(async () => {
-      const runtimeDetails = currentRuntime ? await Ajax().Runtimes.runtimeV2(workspaceId, currentRuntime.runtimeName).details() : null;
+    const announceModalOpen = async () => {
       Ajax().Metrics.captureEvent(Events.cloudEnvironmentConfigOpen, {
         existingConfig: !!currentRuntime,
         ...extractWorkspaceDetails(workspace.workspace),
       });
+    };
+    announceModalOpen();
```

Comment:
This wrapper function looks unnecessary. Generally, you see wrapper functions in `useEffect` / `useOnMount` effect functions because the effect function can't be `async` (effect functions must return void or a cleanup function, not a Promise). But since `captureEvent` isn't awaited, the effect function doesn't need to be `async`.

```suggestion
    Ajax().Metrics.captureEvent(Events.cloudEnvironmentConfigOpen, {
      existingConfig: !!currentRuntime,
      ...extractWorkspaceDetails(workspace.workspace),
    });
```